docs/quick-start.md

```diff
@@ -0,0 +1,92 @@
+# Quick Start Guide
+
+## Overview
+
+This guide is designed to get you up and running working with the Jetpack Monorepo quickly following recommended and supported guidelines. For more detailed information, including alternate local dev environments, running unit tests, best coding practices, and more, you can the [full Development Environment guide here](docs/development-environment.md#clone-the-repository). 
+
+## Installation
+
+Prior to installation, we recommend using [`Homebrew`](https://brew.sh/) to manage installations and [`nvm`](https://github.com/nvm-sh/nvm/) to manage Node.js versions. If you don't already have those installed, you can do so by copy/pasting each of the following commands and running them in your terminal:
+
+- Homebrew: `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+- nvm: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash`
+
+The Jetpack Monorepo requires the following to be installed on your machine:
+
+- Node.js (used for build process and our CLI): `nvm install`
+	- You can see the current latest supported version of Node.js in the monorepo root's [`package.json`](https://github.com/Automattic/jetpack/blob/master/package.json) under `engines`
+	- You can switch versions of node using nvm: `nvm ls` to list installed versions of node, `nvm use [version number]` to switch.
+- PNPM (a Node.js package manager): `npm install -g pnpm`
+- Composer (our PHP package manager): `brew install composer`
+- PHP (the language at the core of the WordPress ecosystem): `brew install php@8.0`
+- Once the pre-requisites are installed, you can clone the GitHub repo:
+	- `git clone https://github.com/Automattic/jetpack.git`
+	- If you're not an Automattician, you can [fork the repo following the instructions here](https://docs.github.com/en/get-started/quickstart/contributing-to-projects).
+- Jetpack CLI (an internal tool that assists with development): `pnpm install && pnpm cli-setup`
+	- [You can read more about using the CLI here](https://github.com/Automattic/jetpack/blob/master/tools/cli/README.md).
+
+## Running Jetpack locally
+
+After everything is installed, you're ready to run Jetpack locally! While there are other supported methods of doing this, we recommend and support using Docker containers. 
+
+To setup Docker:
+- Run `brew install --cask docker` or [download the latest version here](https://www.docker.com/products/docker-desktop). 
+- Copy the settings file so you can modify passwords and other configurations: `cp tools/docker/default.env tools/docker/.env`
+- Start the Docker container using `jetpack docker up`
+- Install WordPress in your Docker container using `jetpack docker install` 
+- Open up http://localhost to see your site!
+
+For more in depth docker instructions, follow the [Docker environment for Jetpack Development guide](../tools/docker/README.md).
+
+### Setting up Jurassic Tube
```

Comment:
I have to try these pieces again myself, but I definitely feel confused by why you would use a local jetpack setup and not use JT to get a link to wpcom. It feels like this should just be one more step in setting up Docker.

I think this whole section should be combined with the one above. Setting up JT should be required to set someone up. They probably don't know whether they need the "network related features of Jetpack" or not.

Reply:
IMO the main reason to call it out is that JT is one of the few things that you can't use if you're not an Automattician.

The other thing mentioned in this doc is that you need to fork the repo to be able to submit PRs, because you won't be able to push PR branches directly to Automattic/jetpack.

Reply:
Yeah, I think leaving them separate for now is a bit cleaner. We can consolidate in the future especially if we end up adding the scripts as part of the monorepo. I tried to clarify the description of that section a bit more.

Reply:
That makes sense and the update is helpful. thanks!